src/screens/NFTList.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { t } from 'ttag'
+import BackButton from '../components/BackButton';
+import { NFT_LIST_PER_PAGE } from '../constants';
+import HathorPaginate from '../components/HathorPaginate';
+import NFTListElement from '../components/NFTListElement';
+import helpers from '../utils/helpers';
+import { connect } from "react-redux";
+
+
+const mapStateToProps = (state) => {
+  const registeredTokens = state.tokens;
+  const tokenMetadata = state.tokenMetadata || {};
+  const nftList = Object.values(tokenMetadata).filter((meta) => {
+    const hasNFTMedia = meta.nft_media && meta.nft_media.file;
+    const isRegistered = state.tokens.find((registeredToken) => registeredToken.uid === meta.id);
+    return hasNFTMedia && isRegistered;
+  });
+
+  const nftData = nftList.map((meta) => {
+    const tokenConfig = state.tokens.find((registeredToken) => registeredToken.uid === meta.id);
+    const balance = state.tokensBalance[meta.id] || { available: 0 };
+    return {
+      name: tokenConfig.name,
+      symbol: tokenConfig.symbol,
+      balance: balance.available,
+      ...meta,
+    };
+  });
+
+  return {
+    nftData,
+  };
+};
+
+/**
+ * List all NFTs with explorer asset
+ *
+ * @memberof Screens
+ */
+class NFTList extends React.Component {
+  /**
+   * page: {Number} Current page of the list
+   * totalPages: {Number} Total number of pages of the list
+   */
+  state = {
+    page: 1,
+    totalPages: 0,
+  }
+
+  componentDidMount() {
+    this.setState({ totalPages: this.getTotalPages() });
+  }
+
+  /**
+   * Return total number of pages of the list
+   *
+   * @return {Number} Total number of pages of the list
+   */
+  getTotalPages = () => {
+    return Math.ceil(this.props.nftData.length / NFT_LIST_PER_PAGE);
+  }
+
+  /**
+   * Event received from pagination component after a page button in clicked
+   *
+   * @param data {Object} Data with clicked page {'selected'}
```

Comment:
Add a note that it is 0-indexed (while `this.state.page` is 1-indexed). Maybe we should add a note about `this.state.page` right above its definition as well.

Reply:
Done.